coalib/coala.py

```diff
@@ -31,6 +31,7 @@ def main():
         # Note: We parse the args here once to check whether to show bears or
         # not.
         args = default_arg_parser().parse_args()
+        console_printer = ConsolePrinter(print_colored=args.no_color)
```

Comment:
no_color should be a const store `True` instead of false, and I would give `not args.no_color` here, that makes way more sense to me